types/registry.go

```diff
@@ -0,0 +1,66 @@
+package types
+
+import "time"
+
+// CompanyAddress represents a company address
+type CompanyAddress struct {
+	AdressLine1 string `json:"address-line-1"`
```

Comment:
Small typo, `AdressLine1` should be `AddressLine1`  Ideally we use `address-line1` for the JSON tag instead of `address-line-1`

Reply:
Man, I really have problems writing address :). Fixed